### ISSUE TITLE
Feature request: Mark as read later

### ISSUE THREAD

**Issue by mat-l** [Huessenbergnetz/ocNews#68]
I'd like to ask about a new feature in ocnews. Would it be possible to mark the read articles directly locally as read and try to sync them later?

Due to the sometimes slower connection (e.g. owncloud or nextcloud is running on Raspberry Pi) the marking as read need 2-4 seconds. While then already switching to the next article, ocnews fails to mark it as read. Later on you read the article sometimes again and again which is a bit annoying.
Is it possible to add this feature? 
Best matl

**Comment by buschmann23** [comment]
Hey, on which OS are you using ocNews? If you are using the SailfishOS version, there is a successor called [Fuoten](https://github.com/Buschtrommel/Fuoten) that is available as preview version on [OpenRepos](https://openrepos.net/content/buschtrommel/fuoten-beta).

**Comment by mat-l** [comment]
Yeah using SailfishOS on Jolla C. Will check the preview versions. Great thanks. :)
